src/amo/components/AddonVersionCard/styles.scss

```diff
@@ -1,24 +1,43 @@
 @import '~amo/css/styles';
 
 .AddonVersionCard {
-  display: flex;
+  @include respond-to(large) {
+    display: flex;
+  }
 
   .InstallButtonWrapper {
     align-self: center;
   }
 }
 
 .AddonVersionCard-header {
-  @include font-bold();
+  @include font-regular();
   @include text-align-start();
 
-  font-size: $font-size-default;
+  font-size: 24px;
+}
```

Comment:
So we don't have a similar font size variable already?

Reply:
Hmm, there is a `font-size-l` in `src/ui/css/vars.scss` which is defined as `24px`, but there is also a `font-size-l` in `src/core/css/inc/vars.scss` which is defined as `28px`, and the latter is what seems to be taking effect in this case.

There's also a size in the latter file called `$font-size-form-header` which is `24px`, but I wasn't sure if that was really an accurate name to use for this. I guess it is a form of header, so it's probably okay. I'll switch from an explicit `24px` to `$font-size-form-header`.